src/main/java/core/basesyntax/dao/ma/MentorDaoImpl.java

```diff
@@ -2,7 +2,9 @@
 
 import core.basesyntax.model.ma.Mentor;
 import java.util.List;
+import org.hibernate.Session;
 import org.hibernate.SessionFactory;
+import org.hibernate.query.Query;
 
 public class MentorDaoImpl extends PersonDaoImpl implements MentorDao {
```

Comment:
how about refactoring `save()` method in PersonDaoImpl using transaction?

Reply:
done